bindings/python/examples/client/post_raw_block.py

```diff
@@ -11,4 +17,4 @@
 result = client.post_block_raw(blockBytes)
 
 # Print block raw
-print(f'{result}')
+print(f'Posted raw block: {os.environ["EXPLORER_URL"]}/block/{result}')
```

Comment:
Does this need to be this?
```suggestion
print(f'Posted raw block: {os.environ["EXPLORER_URL"]}/block/{result[0]}')
```

Reply:
no, just a single BLockId returned